integration/terraform/ec2/linux/centOS8-setup.sh

```diff
@@ -0,0 +1,22 @@
+# You can use this script to install all required deps for integration testing to a centos8 based linux (rhel8 ol8)
```

Comment:
Where does this setup script get called from?

Reply:
No where. These are the commends I used to add the required deps to an ami before building the image.

Reply:
I take an ami lets say rhel8  -> create an ec2 instance -> add these deps -> build a new ami that I use in the test